.appveyor.yml

```diff
@@ -0,0 +1,20 @@
+environment:
+  matrix:
+    - nodejs_version: '8'
+    - nodejs_version: '8'
```

Comment:
This line looks like a duplicate? Unless the intention was to test Node 8.x twice.

Reply:
😬 🤦‍♂️